node-graph/gcore/src/vector/generator_nodes.rs

```diff
@@ -72,11 +73,11 @@ fn rectangle<T: CornerRadius>(
 }
 
 #[node_macro::node(category("Vector: Shape"))]
-fn regular_polygon<T: AsU64>(
+fn regular_polygon<T: AsU64 + std::cmp::PartialOrd + FromPrimitive>(
```

Comment:
@TrueDoctor can we avoid needing to include these trait bounds for `T` here?

Reply:
Yes I described the solution when this was discussed originally, I still might have the partial code for that somewhere. These bounds should basically automatically be added by the macro

Reply:
But we can also add that as a todo and implement that later if there are enough users that not having it becomes annoying

Reply:
Fixed in 1d361c030b7058bc2d956547778d92d0b26bfc23, if the code is correct. (@TrueDoctor)